src/guides/v2.3/frontend-dev-guide/themes/theme-images.md

```diff
@@ -172,7 +172,7 @@ Check the following example on getting the breakpoints variable by the [`Gallery
  */
 public function getBreakpoints()
```

Comment:
Hi @BrentRobert. In the following example, we have a piece of code from the actual codebase (`/app/code/Magento/Catalog/Block/Product/View/Gallery.php`) as explains the sentence before the example. 

Unfortunately, the mentioned piece of code still uses the `encode` method 

https://github.com/magento/magento2/blob/2.3/app/code/Magento/Catalog/Block/Product/View/Gallery.php#L127

So we cannot simply reference the existing method and change only the documentation.

Reply:
That's my bad, didn't see that.